packages/maskbook/src/plugins/Wallet/services/keystore.ts

```diff
@@ -1,6 +1,8 @@
 import scrypt from 'scrypt-js'
 import Web3Utils from 'web3-utils'
 import { Buffer } from 'buffer'
+import keystoreSchema from './keystore.schema.json'
+import z_schema from 'z-schema'
```

Comment:
??? @Jack-Works

Reply:
We're using z-schema to verify if an JSON object satisfy a JSON Schema